examples/beego/main.go

```diff
@@ -0,0 +1,49 @@
+
+
+import (
+	"github.com/astaxie/beego"
+	"github.com/ulule/limiter"
+	"github.com/ulule/limiter/drivers/store/memory"
+	"github.com/astaxie/beego/context"
+	"net/http"
+	"strconv"
+)
+
+var rateLimiter *limiter.Limiter
+
+func main() {
+	rate, err := limiter.NewRateFromFormatted("2-H")
+	if err != nil{
+		panic(err)
+	}
+	store := memory.NewStore()
+
+	rateLimiter = limiter.New(store, rate)
+
+	//More on Beego filters here https://beego.me/docs/mvc/controller/filter.md
+	beego.InsertFilter("*", beego.BeforeRouter, rateLimit)
```

Comment:
Could you provide a high-order function to avoid this global variable. 
Thank you.

Reply:
Updated!